include/pluginplay/printing/mermaid.hpp

```diff
@@ -15,6 +15,7 @@
  */
 
 #pragma once
+#include "module_manager/module_manager_class.hpp"
```

Comment:
```suggestion
#include <pluginplay/module_manager/module_manager_class.hpp>
```
Long story short, if a header file lives in the `include/` directory you should include it with angle brackets (and start the path with the directory which follows `include/` in the path, which in this case is `pluginplay`)